ci-operator/config/openshift/ols-load-generator/openshift-ols-load-generator-main.yaml

```diff
@@ -15,98 +15,148 @@ releases:
     candidate:
       product: ocp
       stream: nightly
-      version: "4.18"
+      version: "4.15"
 resources:
   '*':
     requests:
       cpu: 100m
       memory: 200Mi
 tests:
 - as: ols-load-test-10workers
-  cron: 0 0 1-7,15-21 * 1
+  cron: 0 0 * * *
   steps:
     allow_skip_on_success: true
-    cluster_profile: aws-perfscale
+    cluster_profile: aws-perfscale-qe
     env:
-      BASE_DOMAIN: perfscale.devcluster.openshift.com
+      BASE_DOMAIN: qe.devcluster.openshift.com
       COMPUTE_NODE_REPLICAS: "3"
       COMPUTE_NODE_TYPE: m6i.xlarge
+      ES_BENCHMARK_INDEX: ols-load-test-results*
+      ES_METADATA_INDEX: perf_scale_ci*
+      ES_TYPE: qe
+      HUNTER_ANALYZE: "true"
+      LOOKBACK: "180"
+      LOOKBACK_SIZE: "15"
       OLS_TEST_DURATIONS: 5m,10m,20m
       OLS_TEST_WORKERS: "10"
       OPENSHIFT_INFRA_NODE_INSTANCE_TYPE: r5.xlarge
+      ORION_CONFIG: examples/ols-load-generator.yaml
```

Comment:
@vishnuchalla we are not suing the ORION_CONFIG - just the ORION_ENVS is getting used ?

Reply:
Both are getting used. step-registry has the shell script that describes their usage.